packages/store-sdk/src/analytics/wrap.ts

```diff
@@ -40,34 +40,47 @@ export type AnalyticsEvent =
   | SignupEvent
   | ShareEvent
 
-export type WrappedAnalyticsEvent<T extends AnalyticsEvent> = {
+export interface UnknownEvent {
+  type: string
+  data: unknown
+}
+
+export type WrappedAnalyticsEventData<T extends UnknownEvent> = Omit<
+  T,
+  'type'
+> & {
+  // Sadly tsdx doesn't support typescript 4.x yet. We should change this type to this when it does:
+  // type: `store:${T['type']}`
```

Comment:
we are usig 4.x already, aren't we? 👀

Reply:
We are, but tsdx isn't. It's confusing, but essentially the build fails if we use these kinds of types introduced on 4.x. Look at the commit before the last one, it has the error.